bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -72,6 +72,7 @@
     long endEntryId;
     long requestTimeNanos;
     OpStatsLogger readOpLogger;
+    OpStatsLogger readOpDmLogger; // Track Under replication detected at read.
```

Comment:
is `counter` enough for this purpose?

the reason I am asking this, `OpStatsLogger` will introduce multiple "metrics". If we only care about counts, we should just use "Counter". Use "OpStatsLogger" when you care about percentile latencies.